spec/classes/mod/php_spec.rb

```diff
@@ -290,9 +290,8 @@
       end
 
       # all the following tests are for legacy php/apache versions. They don't work on modern ubuntu and redhat 8
-      next if (facts[:os]['release']['major'].to_i > 15 && facts[:os]['name'] == 'Ubuntu') ||
```

Comment:
I think this should be simplified to still test for Ubuntu, just like Debian.

Reply:
The tests were skipped for those and I updated the code below to match the Debian OS family. The logic should be the same now.

Reply:
Ah yes, I didn't read it carefully enough. Feels like we can also apply the same tricky to the Red Hat OS family below, but that can be done in another PR. Overall I think using `next` to skip tests below something is a poor practice. Indenting is much more reliable IMHO.